source/js/openStreetMap.ts

```diff
@@ -99,27 +66,33 @@ class OpenStreetMap {
         }
     }
 
-    initialize(map) {
-        const markerElementPairs = createMarkerElementPairs(map, this.markers, this.container);
+    initialize(map: LeafletMap) {
+        const AddMarkersInstance = new AddMarkers(map, this.markers as MarkerClusterGroup, this.container);
+        const markerElementPairs = AddMarkersInstance.markerElementObjects();
```

Comment:
Renamed everything (including the interface) to markerElementObjects. There was a function with the same name, since it only returns the array of markerElementObjects this was renamed to "getMarkerElementObjects"